### PR TITLE
APC sound fixes and some electrical code cleanup

### PR DESCRIPTION
**Pull request by Aranclanos**
### Purpose
Changes the APC powerless sound from playing when the battery stops being charged, now it'll only play when the APC runs out of power.
Fixes ACPs from playing the powerless sound at roundstart.
Cleans up a large chunk of electric code, mainly for readability.

### CODE REVIEW

File: UnityProject/Assets/Scripts/Objects/Engineering/PowerGenerator.cs [hunk 1]
```diff
@@ -61,13 +61,14 @@ void Awake()
...
 		{
+
 			var itemStorage = GetComponent<ItemStorage>();
 			itemSlot = itemStorage.GetIndexedItemSlot(0);
 			securable.OnAnchoredChange.AddListener(OnSecuredChanged);
 			if (startAsOn)
 			{
 				fuelAmount = fuelPerSheet;
-				TryToggleOn();
+				//TryToggleOn();
```
Comment (Bod9001) [inline UnityProject/Assets/Scripts/Objects/Engineering/PowerGenerator.cs]: accidentally left this in?

File: UnityProject/Assets/Scripts/Systems/Electricity/Electrical processes/ElectricalPool.cs [hunk 2]
```diff
@@ -1,22 +1,28 @@
-using System.Collections;
 using System.Collections.Generic;
-using UnityEngine;
 
 public class ElectricalPool
 {
-	public static void PoolsStatuses()
```
Comment (Bod9001) [inline UnityProject/Assets/Scripts/Systems/Electricity/Electrical processes/ElectricalPool.cs:7]: was useful for debug

Comment (Bod9001) [reply UnityProject/Assets/Scripts/Systems/Electricity/Electrical processes/ElectricalPool.cs:7]: ./checking for leaks

File: UnityProject/Assets/Scripts/Objects/Engineering/PowerGenerator.cs (hunk 1 above)
Comment (Aranclanos) [reply UnityProject/Assets/Scripts/Objects/Engineering/PowerGenerator.cs]: goo catch, fixed

File: UnityProject/Assets/Scripts/Systems/Electricity/Electrical processes/ElectricalPool.cs (hunk 2 above)
Comment (Aranclanos) [reply UnityProject/Assets/Scripts/Systems/Electricity/Electrical processes/ElectricalPool.cs:7]: added back

**Pull request merged**